astro/src/content/config.js

```diff
@@ -66,10 +72,13 @@ const docsCollection = defineCollection({
 const blogCollection = defineCollection({
   schema: z.object({
     title: z.string().refine(
-        title => !title.endsWith('.'),
-        { message: "Title cannot end with a period (.)" }
+        title => !/[.]$/.test(title),
```

Comment:
why regex instead of normal string operators?

Reply:
ah, because I was using more than one punctuation mark for the title check, until I realized that we sometimes have an exclamation point in titles.

Reply:
Can revert if you feel strongly.